lib/Target/AVR/AVRInstrInfo.h

```diff
@@ -107,6 +107,12 @@ class AVRInstrInfo : public AVRGenInstrInfo {
 
   bool isBranchOffsetInRange(unsigned BranchOpc,
                              int64_t BrOffset) const override;
+
+  unsigned insertIndirectBranch(MachineBasicBlock &MBB,
+                                MachineBasicBlock &NewDestBB,
+                                const DebugLoc &DL,
+                                int64_t BrOffset = 0,
```

Comment:
I would avoid specifying defaults here so there's no risk we get out of sync with upstream LLVM.

I doubt that there will any by callers of `AVRInstrInfo::insertIndirectBranch` directly - the only caller will likely be outside of `lib/Target/AVR` and thus will automatically see the default values because it will be working through a `TargetInstrInfo*` object.